components/notifications-service/server/lib/formatters/compliance_helper.ex

```diff
@@ -99,4 +102,32 @@ defmodule Notifications.Formatters.ComplianceHelper do
   end
   # Anything that's not a failure gets excluded
   defp prune_results([_| results], acc), do: prune_results(results, acc)
+
+  @spec get_servicenow_compliance_notification(ComplianceFailure.t):: map()
+  def get_servicenow_compliance_notification(notification) do
+    get_servicenow_compliance_notification(notification, @crit_control_threshold)
+  end
+  def get_servicenow_compliance_notification(%ComplianceFailure{test_totals: totals} = notification, control_threshold) do
```

Comment:
The `get_servicenow_compliance_notification` functions should be moved into the servicenow.compliance.ex file.

Reply:
I think I see now why the `get_servicenow_compliance_notification` function was added here. Because of the 'servicenow.compliance.ex' and the 'servicenowcritical.compliance.ex' files both need it. 🤔 

Maybe remove the 'servicenowcritical.compliance.ex' file and add the code to the 'servicenow.compliance.ex' file. You could have two 'format' functions. One 'format_non_critical' and 'format_critical'. Then move this function back to this file also.

Reply:
Done